docs/application/native/guides/ui/dali/progressbar-n.md

```diff
@@ -0,0 +1,120 @@
+# ProgressBar
+
+The progress bar is a control to give the user an indication of the progress of an operation. The base class for the progress bar control is `Dali::Toolkit::ProgressBar`.
+
+In this tutorial, the following subjects are covered:
+
+[ProgressBar events](#1)<br>
+[Creating a ProgressBar](#2)<br>
+[ProgressBar Properties](#3)<br>
+[Customizing the ProgressBar Appearance](#4)<br>
+
+## Overview
+
+The ProgressBar has two modes to represent the progress. The default mode is Determinate mode.
+
+- **Determinate mode** is to show a specific quantity of progress occurred. If you want to set the amount of progress, then set `PROGRESS_VALUE` property. If not supplied, the default value is 0.
+
+ *Figure: Determinate progress bar*
+ ![Determinate mode](./media/determinated_progress.png)
+
+
+- **Indeterminate mode** is to show the progress without a specific amount of progress indicated. If you want to indicate an indeterminate progress bar, then set `INDETERMINATE` enabled. You can also show an animation using `INDETERMINATE_VISUAL_ANIMATION` property.
```

Comment:
Rephrase as 
- **Indeterminate mode** shows the progress without any specific percentage of progress indicated. If you want to indicate an indeterminate progress bar, then set the INDETERMINATE mode to enabled. You can also show an animation using the INDETERMINATE_VISUAL_ANIMATION property.

Reply:
Done.